test/JDBC/expected/BABEL-5119_before_16_5-vu-verify.out

```diff
@@ -110,17 +146,16 @@ babel5119_v3#!#CREATE VIEW BABEL5119_v3 AS SELECT 1
 -- General crossdb queries
 SELECT tb_1.*, tb_2.* FROM BABEL5119_db.dbo.BABEL5119_t1 tb_1, master.guest.BABEL5119_t4 tb_2
 GO
-~~START~~
-int#!#int
-~~END~~
+~~ERROR (Code: 33557097)~~
+
+~~ERROR (Message: relation "guest.babel5119_t4" does not exist)~~
 
 
 SELECT vw_1.*, vw_2.* FROM BABEL5119_db.dbo.BABEL5119_v1 vw_1, master.guest.BABEL5119_v4 vw_2
```

Comment:
Lets modify these tests to show the dbo objects. 
```
SELECT vw_1.* FROM BABEL5119_db.dbo.BABEL5119_v1 vw_1
```
There are other examples in this file too.